tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+temp/
+test_app/
```

Comment:
I don't understand why these directories should be here because normally they should be created before the tests and removed after the tests. If they exist after end of the tests it is a bug.